admin/src/react-components/account-edit-toolbar.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { Toolbar, SaveButton } from "react-admin";
+import { withStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import { Dialog, DialogContent, DialogContentText, DialogActions } from "@material-ui/core";
+
+const accountEditToolbarStyles = {
+  spaceBetween: { justifyContent: "space-between" },
+  dialogActions: { padding: "0 5px" }
+};
+
+const deleteStates = {
+  confirming: "confirming",
+  deleting: "deleting",
+  succeeded: "succeeded",
+  failed: "failed"
+};
+
+export const AccountEditToolbar = withStyles(accountEditToolbarStyles)(props => {
+  const { classes, ...other } = props;
```

Comment:
i'd through "id" in here as well so you dont have to do prop.id

Reply:
Wouldn’t that change the value of `...other`, which is passed to the Toolbar component?

Reply:
yea let's ignore this for now